trie/database.go

```diff
@@ -332,6 +343,7 @@ func (db *Database) insert(hash common.Hash, size int, node node) {
 		node:      simplifyNode(node),
 		size:      uint16(size),
 		flushPrev: db.newest,
+		commited:  false,
```

Comment:
`committed` as well

Reply:
okay, just want to make it explicitly. By the way, I will remove it if it's out favor.